examples/demo-app/src/cloud-providers/index.js

```diff
@@ -22,15 +22,30 @@ import {AUTH_TOKENS} from '../constants/default-settings';
 
 import DropboxProvider from './dropbox/dropbox-provider';
 import CartoProvider from './carto/carto-provider';
+import FsqStudioProvider from './fsq-studio/fsq-studio-provider';
+
+const {
+  DROPBOX_CLIENT_ID,
+  CARTO_CLIENT_ID,
+  FSQ_STUDIO_CLIENT_ID,
+  FSQ_STUDIO_DOMAIN,
+  FSQ_STUDIO_API_URL,
+  FSQ_STUDIO_USER_MAPS_URL
+} = AUTH_TOKENS;
```

Comment:
This is no longer just AUTH_TOKENS. To make the code easier to follow, maybe consider renaming to `CLOUD_PROVIDERS_CONFIGURATION` or similar?

Reply:
yeah we can do that